### ISSUE TITLE
How to test performance with 3 channels (R,G,B) on lfw?

### ISSUE THREAD

**Issue by PlumedSerpent** [ly-atdawn/LightCNN-gulon#1]
I tried to training a model with 3 channels image, so I change the parameter "image_shape" in train_lightcnn.py . After 35 epochs, it reached to convergence on MS-celeb1M_10K, and got pretty impressive accuracy (Train acc 99.9%,Test acc 96.8%). But when I test it on lfw, it got a poor performance(only about 60%).
I've noticed your comments in "extract_features.py" .
![image](https://user-images.githubusercontent.com/11538323/44586824-9db37a00-a7e3-11e8-9176-cef32b74d9c2.png)
So I tried following imread methods,but all of them leads to similar results.
Method1:
    img = cv2.imread(image_path,1)
    img = cv2.resize(img, (128, 128)) # resize
    b,g,r = cv2.split(img) 
    img_rgb = cv2.merge([r,g,b]) 
    img_rgb = img_rgb - np.array([123.68,116.779,103.939])
    img_rgb = np.reshape(img_rgb, (1, 3, 128, 128))

Method2:
    img = cv2.imread(image_path, 1)
    img = img - [np.sum(img) / img.size]
    img = np.reshape(img, (1, 3, 128, 128))

Could you please apply some suggestion？Did I make some mistake in my implementation？

**Comment by lyatdawn** [comment]
Sorry, my reply is so late.
I think, when train LightCNN model, we use the imag is a gray image, so when we use the pretrained model to extract the feature, the image should be a gray imag. i.e.:
img = cv2.imread(image_path, 0)
